Apps/Src/common.c

```diff
@@ -9,3 +9,11 @@ void print_float(char * str, float f) {
     int32_t d = (f<0)?-f:f;
     printf("%d.%02d\n\r", (int)n, (int)d);
 }
+
+void print_bin(char * str, uint32_t i){
+    if(str) printf(str);
```

Comment:
`printf` should always take a string literal, maybe write this as `printf("%s", str);`

Reply:
Got it!